.pre-commit-config.yaml

```diff
@@ -38,7 +38,7 @@ repos:
         entry: mypy
         language: python
         types: [python]
-        args: [--scripts-are-modules]
+        args: [--scripts-are-modules, --warn-unused-ignores]
```

Comment:
Could these also be set in the `mypy.ini` file instead? I think it'd make sense to keep all configuration in one place.

Reply:
yep good point, done